tests/ha/test_ha_cluster_health_gui.py

```diff
@@ -188,10 +188,9 @@ def test_cluster_one_by_one_os_shutdown_gui(self):
             node_name = self.srvnode_list[node]
             LOGGER.info("Verify if node state online")
             self.ha_gui_obj.verify_node_state(node, "online")
-            LOGGER.info(f"Shutting down {node_name}")
+            LOGGER.info("Shutting down %s", node_name)
```

Comment:
same here. TO disable too-many-statements before test_cluster_one_by_one_os_shutdown_gui